packages/ckeditor5-engine/src/view/domconverter.js

```diff
@@ -1644,8 +1661,15 @@ function hasBlockParent( domNode, blockElements ) {
  * {@glink framework/guides/architecture/editing-engine#editing-pipeline editing pipeline} of the editor. To avoid this,
  * the `<script>` element was renamed to `<span data-ck-unsafe-element="script"></span>`.
  *
- * @error domconverter-unsafe-element-detected
- * @param {module:engine/model/element~Element|HTMLElement} unsafeElement The editing view or DOM element
+ * @error domconverter-unsafe-script-element-detected
+ * that was renamed.
+ */
+
+/**
+ * The {@link module:engine/view/domconverter~DomConverter} detected a `<style>` element that may affect the editing experience.
```

Comment:
"While rendering the editor content, the ..."

"To avoid this, the `<style>` element was replaced with ..."

Reply:
Fixed.